### DISCUSSION TITLE
How to set multiple colors in marker label?

### DISCUSSION THREAD

**Discussion by csbenz** [patrykandpatrick/vico#548] [Questions] [answered]
How to set multiple colors in marker label?

**Comment by csbenz** [comment]
In views, not Compose

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can use a custom `MarkerLabelFormatter` and Android’s `SpannableString` to do this. `SpannableString` implements `CharSequence`, so instances of it can be returned from `MarkerLabelFormatter#getLabel`. `MarkerLabelFormatter`s are applied to `MarkerComponent`s via the `labelFormatter` field.
